File: order_backend_springboot/src/main/java/com/order/domain/entity/Category.java
package com.order.domain.entity;


import lombok.Data;
import org.hibernate.annotations.DynamicUpdate;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.util.Date;

@Entity
@DynamicUpdate
@Data
public class Category {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String name;

    private String description;

    private Integer weight = 0;

    private Integer status = 0;

    private Date createTime;

    private Date updateTime;

    private Integer del = 0;
}

File: order_backend_node/app/model/category.js
'use strict';

module.exports = app => {
  const { INTEGER, STRING } = app.Sequelize;
  return app.model.define('category', {
    id: { type: INTEGER, primaryKey: true, autoIncrement: true },
    name: STRING,
    description: STRING,
    weight: INTEGER,
    status: INTEGER,
    del: INTEGER,
  }, {
    freezeTableName: true,
  });
};
